templates/main_menu.html

```diff
@@ -18,47 +18,48 @@ <h1 class="menu-page-header">
             <span>Layouts</span>
         </li>
         <li class="selected">
-            <a {% url 'management:dashboard' %}>
+            <a href="{% url 'management:dashboard' %}">
                 <div class="icon-w">
                     <div class="os-icon os-icon-layout"></div>
                 </div>
                 <span>Dashboard</span>
             </a>
         </li>
-        <li>
-            <a {% url 'management:dashboard' %}>
-                <div class="icon-w">
-                    <div class="os-icon os-icon-layers"></div>
-                </div>
-                <span>Menu Styles</span>
-            </a>
-        </li>
+        {#        <li>#}
+        {#            <a href="{% url 'management:dashboard' %}">#}
+        {#                <div class="icon-w">#}
+        {#                    <div class="os-icon os-icon-layers"></div>#}
+        {#                </div>#}
+        {#                <span>Menu Styles</span>#}
+        {#            </a>#}
+        {#        </li>#}
         <li class="sub-header">
-            <span> Client </span>
+            <span> Company </span>
         </li>
         <li>
-            <a href="{% url 'management:create_client' %}">
+            <a href="{% url 'management:create_company' %}">
                 <div class="icon-w">
                     <div class="os-icon os-icon-package"></div>
                 </div>
-                <span> Create Client Profile </span>
+                <span> Create Company Profile </span>
             </a>
         </li>
         <li>
-            <a href="{% url 'management:create_client' %}">
+            <a href="{% url 'management:list_companies' %}">
                 <div class="icon-w">
                     <div class="os-icon os-icon-file-text"></div>
                 </div>
-                <span> Edit Client Profile </span>
-            </a>
-        </li>
-        <li>
-            <a href="{% url 'management:view_client' %}">
-                <div class="icon-w">
-                    <div class="os-icon os-icon-life-buoy"></div>
-                </div>
-                <span> View Client Profile </span>
+                <span> Companies List </span>
             </a>
         </li>
+        {# to be discussed #}
```

Comment:
what you want to discuss

Reply:
There was another category  "View Company" below "List Companies", which I think shouldn't be there.